test/e2e/windows/vhd.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package windows
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"text/template"
+)
+
+// VHD is a Windows Virtual Hard Disk, VHD generates powershell commands
+// that are used by HostExec to interact with VHD created in a remote node
+type VHD struct {
+	Path      string
+	SizeBytes int
+}
+
+// NewVHD creates a new VHD
+func NewVHD(path string, sizeBytes int) *VHD {
+	return &VHD{
+		Path:      path,
+		SizeBytes: sizeBytes,
+	}
+}
+
+// StageScript generates a script to initialize a VHD
+func (vhd *VHD) StageScript() (string, error) {
```

Comment:
what is the return string?

Reply:
this file generates scripts to be used by a windows executor, the executor works with strings